api/authorizers/build.js

```diff
@@ -1,25 +1,26 @@
-const { User, Site } = require("../models")
+const { User, Site } = require('../models');
 
-const authorize = (user, build) => {
-  return User.findById(user.id, { include: [Site] }).then(user => {
-    for (site of user.Sites) {
-      if (build.site || build.Site.id === site.id) {
```

Comment:
This was the bad if-condition. Note it was checking just if `build.site` was truthy.